device-base/edison/jessie/entry.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+HOSTNAME=$(cat /etc/hostname)
+echo "127.0.1.1 $HOSTNAME" >> /etc/hosts
+hostname $HOSTNAME
+
+if [ "$INITSYSTEM" = "on" ]; then
+	GREEN='\033[0;32m'
+	echo -e "${GREEN}Systemd init system enabled."
+	env > /etc/docker.env
+
+	echo -e "#!/bin/bash\n exec $@" > /etc/resinApp.cmd
+	chmod +x /etc/resinApp.cmd
+
+	mount -t devtmpfs none /dev
+	mount -t debugfs nodev /sys/kernel/debug
```

Comment:
I think it may be worth having a comment here mentioning why this is mounted. Basically edison libmraa, relies on `/sys/kernel/debug/gpio_debug/gpio` being in the container so that the pin modes can be set/changed